src/k8s/api/v1/bootstrap_config.go

```diff
@@ -29,6 +29,8 @@ type BootstrapConfig struct {
 	// Seed configuration for external certificates
 	CACert                     *string `json:"ca-crt,omitempty" yaml:"ca-crt,omitempty"`
 	CAKey                      *string `json:"ca-key,omitempty" yaml:"ca-key,omitempty"`
+	ClientCACert               *string `json:"client-ca-crt,omitempty" yaml:"client-ca-crt,omitempty"`
```

Comment:
nit: consider adding this new config option here: https://github.com/canonical/k8s-snap/blob/f4fd0e35ac7ee369c9c40213c9cf820b8a2ebe1d/src/k8s/cmd/k8s/k8s_bootstrap_test.go#L35

Reply:
i'd avoid that for now, just because there are more fields missing from that test